cmd/crc/cmd/start.go

```diff
@@ -96,3 +106,26 @@ func validateStartFlags() error {
 	}
 	return nil
 }
+
+func getPullSecretFileContent() (string, error) {
+	// In case user doesn't provide a file in start command or in config then ask for it.
+	if crcConfig.GetString(config.PullSecretFile.Name) == "" {
+		pullsecret, err := input.PromptUserForSecret(fmt.Sprintf("Image pull secret for User (Copy it from %s)", constants.DefaultPullSecretURL))
+		// This is just to provide a new line after user enter the pull secret.
+		fmt.Println()
+		if err != nil {
+			return "", err
+		}
+		return pullsecret, nil
```

Comment:
Should we save this pull secret somewhere? (ideally once we know for sure it's working)

Reply:
@cfergeau then we have to implement a way to check if it is stored somewhere and use it from there. So as per our current flow we ask user to download the pullsecret file and use it as argument or part of config set command so it's already in the filesystem and we don't have to store it.

Reply:
If we only ask it after `crc delete` and on initial start, I would say it's less important to cache it ourselves.